src/renderer/components/kube-object/kube-object-details.tsx

```diff
@@ -169,6 +166,18 @@ export class KubeObjectDetails extends React.Component {
         <item.components.Details object={object} key={`object-details-${index}`} />
       ));
 
+    if (details.length === 0) {
+      const crd = crdStore.getByObject(object);
+
+      /**
+       * This is a fallback so that if a custom resource object doesn't have
+       * any defined details we should try and display at least some details
+       */
+      if (crd) {
```

Comment:
Why do we need to get & check `crd`? `object` already is the crd, right?

Reply:
No object is a resource that is described by a CRD (I would call it a CR).

I guess we should fall back to just `KubeObjectDetails` if `crd` is false.